Lucidtech/Las/Core/Credentials.cs

```diff
@@ -121,37 +121,41 @@ private object JsonDecode(IRestResponse response)
             string clientSecret,
             string authEndpoint,
             string apiEndpoint
-        ) {
+        ) : this()
+        {
             ClientId = clientId;
             ClientSecret = clientSecret;
             AuthEndpoint = authEndpoint;
             ApiEndpoint = apiEndpoint;
-            CommonConstructor();
+            //CommonConstructor();
```

Comment:
remove this comment

Reply:
Can we remove the method CommonConstructor now?

Reply:
Yup - thanks for noticing!